src/assets/toolkit/styles/utils/display.css

```diff
@@ -60,4 +68,16 @@
   .u-lg-block {
     display: block !important;
   }
+
+  .u-lg-inlineBlock {
+    display: inline-block !important;
+  }
+
+  .u-lg-unhideVisually {
```

Comment:
Yep, exactly. The slide count is visually hidden at small screens, but I want to show it at large screens.